pkg/roachpb/api.proto

```diff
@@ -1643,11 +1642,12 @@ message AddSSTableRequest {
   RequestHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
   bytes data = 2;
 
-  // WriteAtRequestTimestamp updates all MVCC timestamps in the SST to the
-  // request timestamp, even if the request gets pushed. This ensures the writes
-  // comply with the timestamp cache and closed timestamp. It also causes the
-  // AddSSTable to be emitted via the range feed, since it respects the closed
-  // timestamp.
+  // SSTTimestampToRequestTimestamp gives the timestamp used for all MVCC
```

Comment:
nit: I see we had some discussion on this comment already, but it is still not ideal. SSTTimestampToRequestTimestamp doesn't rewrite SST itself, is it? Maybe something like:

SSTTimestampToRequestTimestamp is a timestamp used for all MVCC keys in the provided SST. If this timestamp is different from request timestamp (e.g. if the request gets pushed) all timestamps in SST will be rewritten by request timestamp at the time of request processing. This ensures...

Reply:
Yeah, that's better. Updated, with some tweaks.